resources/vue/src/components/site/common/Site.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div>
+        <site-header></site-header>
+        <site-main></site-main>
+        <site-footer></site-footer>
+    </div>
+</template>
+
+<script>
+import SiteHeader from "./SiteHeader";
```

Comment:
I don't think this naming is good, but it's mostly because you don't have the pages/ folder where you can put all of the pages and not mix them with re-usable components.